@ecomplus/storefront-template/template/scss/themes/beauty/_variables.scss

```diff
@@ -0,0 +1,46 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;600&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;600&display=swap');
+
+$icons-font: "line-awesome" !default;
+
+/* Colors: */
+$primary:                  #E83E8C;
```

Comment:
Não pode forçar a cor primária dessa forma, a secundária em alguns temas até que vai, a primária nunca.
Com isso aqui você não permite que a loja utilize a cor da sua marca, uma loja com logo azul teria todos os CTAs rosa por exemplo, seu tema seria utilizável apenas para lojas com cor azul.

Reply:
O que você pode fazer é alterar a cor primária padrão do tema, usando `!default` como acontece em vários temas tipo aí https://github.com/ecomplus/storefront/blob/master/%40ecomplus/storefront-template/template/scss/themes/cerulean/_variables.scss#L33-L40